operator/src/main/resources/scripts/startNodeManager.sh

```diff
@@ -337,13 +337,24 @@ while [ 1 -eq 1 ]; do
     break
   fi
   if [ $((SECONDS - $start_secs)) -ge $max_wait_secs ]; then
-    trace INFO "Trying to put a node manager thread dump in '$nodemgr_out_file'."
-    kill -3 `jps -l | grep weblogic.NodeManager | awk '{ print $1 }'`
+    pid=$(jps | grep NodeManager | awk '{ print $1 }')
+    if [ -z $pid ]; then
+      trace INFO "Node manager process id not found. Cannot create thread dump."
+    else
+      trace INFO "Node manager process id is '$pid'."
+      if [ -x "$(command -v $JAVA_HOME/bin/jcmd)" ]; then
+        trace INFO "Node manager thread dump:"
+        $JAVA_HOME/bin/jcmd $pid Thread.print
```

Comment:
Minor:  The JIRA was old - probably written before the 'kill -3' was added.  How about we try generate a thread dump two ways in case either the new `jcmd` approach or the old `kill -3` method works better? My guess is the answer varies based on the nature of the NM failure.  If you agree, then please add `trace INFO "Trying to put a node manager thread dump in '$nodemgr_out_file'."` followed by `kill -3 $pid` just after the `trace INFO "Node manager process id is '$pid'."` and just before the `if`.

Reply:
I've added the additional dump to the output file using 'kill -3' as I agree that getting thread dumps could also be based on the nature of the NM failure.  It doesn't hurt to have multiple methods for getting thread dumps.